packages/editor/src/lib/hooks/useCanvasEvents.ts

```diff
@@ -74,6 +74,20 @@ export function useCanvasEvents() {
 				})
 			}
 
+			function onPointerEnter(e: React.PointerEvent) {
+				if ((e as any).isKilled) return
+				if (editor.instanceState.isPenMode && e.pointerType !== 'pen') return
+				const canHover = e.pointerType === 'mouse' || e.pointerType === 'pen'
+				editor.updateInstanceState({ isHoveringCanvas: canHover ? true : null })
+			}
+
+			function onPointerLeave(e: React.PointerEvent) {
+				if ((e as any).isKilled) return
+				if (editor.instanceState.isPenMode && e.pointerType !== 'pen') return
+				const canHover = e.pointerType === 'mouse' || e.pointerType === 'pen'
```

Comment:
This will also trigger for certain tablets, such as the Wacom tablets, which come in as mouse rather than pen. There's not much we can do about that, though it's confusing. 

If the logic really is `e.pointerType !== 'touch'` then let's use that instead.

```suggestion
				const canHover = e.pointerType !== 'touch'
```